lciafmt/data/methods.json

```diff
@@ -1,16 +1,20 @@
 [
   {
-    "id": "TRACI 2.1",
+    "id": "TRACI",
```

Comment:
we should go ahead and make this ID TRACI2.1

Reply:
I was trying to align the ID with the Method class, which means we can't have dots in the name. How about TRACI_21 or TRACI_2_1
see [here](https://github.com/USEPA/LCIAformatter/blob/method_storage/lciafmt/__init__.py#L19)